trin-core/src/portalnet/find/query_pool.rs

```diff
@@ -38,10 +38,10 @@ pub trait TargetKey<TNodeId> {
 /// that determines the peer selection strategy, i.e. the order in which the
 /// peers involved in the query should be contacted.
 pub struct QueryPool<TTarget, TNodeId, TResult> {
-    next_id: QueryId,
+    next_id: usize,
```

Comment:
When I merged in the previous `QueryPool` PR I didn't realize that, since the code wasn't used elsewhere in the codebase, rustc/clippy wouldn't throw warnings/errors when things were amiss. 

Once this code became used, the compiler correctly alerted me to the fact that `QueryId` is a struct containing a `usize`, not an alias for `usize`. It makes more sense to me to have it instead be an aliased type for `usize`, and I plan to do that refactor, during which I might realize why `sigp/discv5` declared it this way.